.vscode/cspell.json

```diff
@@ -345,6 +345,7 @@
         "rerank",
         "reranker",
         "sorani",
+        "Vectorize",
```

Comment:
Super minor nit that I am almost embarrassed to comment on but should this be lowercase to match the rest of the entries?